src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/AS_InbreedingCoeff.java

```diff
@@ -0,0 +1,105 @@
+
+import java.util.*;
+
+/**
+ * Allele-specific likelihood-based test for the inbreeding among samples
+ *
+ * <p>This annotation estimates whether there is evidence of inbreeding in a population. The higher the score, the higher the chance that there is inbreeding.</p>
+ *
+ * <h3>Statistical notes</h3>
+ * <p>The calculation is a continuous generalization of the Hardy-Weinberg test for disequilibrium that works well with limited coverage per sample. The output is the F statistic from running the HW test for disequilibrium with PL values. See the <a href="http://www.broadinstitute.org/gatk/guide/article?id=4732">method document on statistical tests</a> for a more detailed explanation of this statistical test.</p>
+ *
+ * <h3>Caveats</h3>
+ * <ul>
+ * <li>The inbreeding coefficient can only be calculated for cohorts containing at least 10 founder samples.</li>
+ * <li>This annotation can take a valid pedigree file to specify founders. If not specified, all samples will be considered as founders.</li>
+ * </ul>
+ *
+ * <h3>Related annotations</h3>
+ * <ul>
+ *     <li><b><a href="https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_annotator_InbreedingCoeff.php">InbreedingCoeff</a></b> outputs a version of this annotation that includes all alternate alleles in a single calculation.</li>
+ *     <li><b><a href="https://www.broadinstitute.org/gatk/gatkdocs/org_broadinstitute_gatk_tools_walkers_annotator_ExcessHet.php">ExcessHet</a></b> estimates excess heterozygosity in a population of samples.</li>
+ * </ul>
+ *
+ */
+//TODO: this can't extend InbreedingCoeff because that one is Standard and it would force this to be output all the time; should fix code duplication nonetheless
+public final class AS_InbreedingCoeff extends InfoFieldAnnotation implements AS_StandardAnnotation {
+
+    public static final int MIN_SAMPLES = 10;
+    private Set<String> founderIds;    //TODO: either use this or enter a bug report
+
+    public AS_InbreedingCoeff(){
+        this(null);
+    }
+
+    public AS_InbreedingCoeff(final Set<String> founderIds){
+        //If available, get the founder IDs and cache them. the IC will only be computed on founders then.
+        this.founderIds = founderIds;
+    }
+
+    @Override
+    public List<String> getKeyNames() { return Collections.singletonList(GATKVCFConstants.AS_INBREEDING_COEFFICIENT_KEY); }
+
+    @Override
+    public List<VCFInfoHeaderLine> getDescriptions() { return Collections.singletonList(GATKVCFHeaderLines.getInfoLine(getKeyNames().get(0))); }
+
+    @Override
+    public Map<String, Object> annotate(final ReferenceContext ref,
+                                        final VariantContext vc,
+                                        final Map<String, PerReadAlleleLikelihoodMap> perReadAlleleLikelihoodMap ) {
+        Utils.nonNull(vc);
+        final HeterozygosityUtils heterozygosityUtils = new HeterozygosityUtils(vc);
```

Comment:
Rename `HeterozygosityUtils` to something that doesn't end in `Utils`, since it looks bizarre to be instantiating a "utils" class like this.